libraries/AP_RPM/AP_RPM.h

```diff
@@ -99,7 +99,7 @@ class AP_RPM
 
     RPM_State state[RPM_MAX_INSTANCES];
     AP_RPM_Backend *drivers[RPM_MAX_INSTANCES];
-    uint8_t num_instances:2;
+    uint8_t num_instances:3;  // This is a bit field not =
```

Comment:
```suggestion
    uint8_t num_instances;
```
Seems you already tripped on this. Why not just make it a full byte to save everyone the headache?